src/Blazored.LocalStorage/LocalStorageService.cs

```diff
@@ -41,7 +46,7 @@ public async Task<T> GetItemAsync<T>(string key)
             if (string.IsNullOrWhiteSpace(serialisedData))
                 return default(T);
 
-            return JsonSerializer.Deserialize<T>(serialisedData);
+            return JsonSerializer.Deserialize<T>(serialisedData, this._jsonOptions);

```

Comment:
Could we drop the `this` keyword.
```suggestion
            return JsonSerializer.Deserialize<T>(serialisedData, _jsonOptions);
```